openedx/features/caliper_tracking/transformers/video_transformers.py

```diff
@@ -4,6 +4,7 @@
 
 import json
 from datetime import timedelta
+
```

Comment:
Is there any unnecessary whitespace added here?

Reply:
There should be a white line between builtin libraries imports and third party packages imports, similarly another line before between current package imports and then similarly with relative imports from repo.

Reply:
Cool.